packages/jupyter-ai/jupyter_ai/chat_handlers/base.py

```diff
@@ -140,6 +145,7 @@ def __init__(
         help_message_template: str,
         chat_handlers: Dict[str, "BaseChatHandler"],
         context_providers: Dict[str, "BaseCommandContextProvider"],
+        message_interrupted: Dict[str, Event],
```

Comment:
Can you assign a default value of `{}` to this parameter to avoid breaking unit tests, for other devs who test subclasses of `BaseChatHandler` in their own repos?